maintenance/countBlogWatchers.php

```diff
@@ -0,0 +1,84 @@
+<?php
+require_once __DIR__ . '/Maintenance.php';
+
+class CountBlogWatchers extends Maintenance {
+
+	const COUNT_BLOG_WATCHERS_EXECUTE = 'CountBlogWatchers::execute';
+
+	public function __construct() {
+		parent::__construct();
+		$this->addOption( 'blog', 'The blog watchlist to recount', true, true );
+		$this->mDescription = 'Count all real subscribers to the watchlist of a seelected blog';
+	}
+
+	public function execute() {
+		if ( !$this->hasOption( 'blog' ) ) {
+			$this->error( 'A blog is required' );
+		}
+
+		$blog = $this->getOption( 'blog' );
+
+		$dbWikia = wfGetDB( DB_SLAVE, [], 'wikia' );
+		$dbWikicities = wfGetDB( DB_SLAVE, [], 'wikicities' );
+		$dbUserPreferences = wfGetDB( DB_SLAVE, [], 'user_preferences' );
```

Comment:
Will it work on prod? I'm asking just to make sure - as I don't see any other usages of 'user_preferences' as a wiki parameter to wfGetDB. Also "slave user_preferences" command works on my devbox but fails on cron-s1